proto/cosmos/base/abci/v1beta1/abci.proto

```diff
@@ -175,3 +176,11 @@ message SearchBlocksResult {
   // List of blocks in current page
   repeated tendermint.types.Block blocks = 6;
 }
+
+// ValidatorUpdate
+message ValidatorUpdate {
+  option (gogoproto.stringer) = true;
+
+  tendermint.crypto.PublicKey pub_key = 1 [(gogoproto.nullable) = false];
+  int64                       power   = 2;
+}
```

Comment:
we only need this type to make it work, should we just depend on comet for it? @aaronc @alexanderbez

Reply:
The pubkey? I think that's fine

Reply:
The valset type entirely

Reply:
If we're defining parts of ABCI ourselves, I would say let's define the entire type, no?

Reply:
since this endblock is tied to comet i may leave it for now.